app/src/main/java/com/example/reflector_android/Models/Article.kt

```diff
@@ -3,7 +3,7 @@ package com.example.reflector_android.Models
 
 // The Article class is a model that represents the data that an Article in the Reflector would have.
 // The input parameters of this article are what are returned when you do an RSS GET request from reflector tab like reflector-online.com/news/
-data class Article(var title: String, var desc: String, var pubDate: String, var link: String, var creator: String) {
+data class Article(val title: String, val desc: String, val pubDate: String, val link: String, val creator: String) {
 
```

Comment:
Delete the extra whitespace below. Do Article(var title...) { } If there is nothing in the class, or remove the {} all together if AndroidStudio lets you.